apps/meteor/server/settings/accounts.ts

```diff
@@ -0,0 +1,949 @@
+			});
+
+			await this.add('Accounts_Default_User_Preferences_omnichannelTranscriptEmail', false, {
+				type: 'boolean',
+				public: true,
+				i18nLabel: 'Omnichannel_transcript_email',
+			});
+		});
+
+		await this.section('Avatar', async function () {
+			await this.add('Accounts_AvatarResize', true, {
+				type: 'boolean',
+			});
+			await this.add('Accounts_AvatarSize', 200, {
+				type: 'int',
+				enableQuery: {
+					_id: 'Accounts_AvatarResize',
+					value: true,
+				},
+			});
+
+			await this.add('Accounts_AvatarExternalProviderUrl', '', {
+				type: 'string',
+				public: true,
+			});
+
+			await this.add('Accounts_RoomAvatarExternalProviderUrl', '', {
+				type: 'string',
+				public: true,
+			});
+
+			await this.add('Accounts_AvatarCacheTime', 3600, {
+				type: 'int',
+				i18nDescription: 'Accounts_AvatarCacheTime_description',
+			});
+
+			await this.add('Accounts_AvatarBlockUnauthenticatedAccess', false, {
+				type: 'boolean',
+				public: true,
+			});
+
+			return this.add('Accounts_SetDefaultAvatar', true, {
+				type: 'boolean',
+			});
+		});
+
+		await this.section('Password_Policy', async function () {
+			await this.add('Accounts_Password_Policy_Enabled', false, {
+				type: 'boolean',
+			});
+
+			const enableQuery = {
+				_id: 'Accounts_Password_Policy_Enabled',
+				value: true,
+			};
+
+			await this.add('Accounts_Password_Policy_MinLength', 7, {
+				type: 'int',
+				enableQuery,
+			});
+
+			await this.add('Accounts_Password_Policy_MaxLength', -1, {
+				type: 'int',
+				enableQuery,
+			});
+
+			await this.add('Accounts_Password_Policy_ForbidRepeatingCharacters', true, {
+				type: 'boolean',
+				enableQuery,
+			});
+
+			await this.add('Accounts_Password_Policy_ForbidRepeatingCharactersCount', 3, {
+				type: 'int',
+				enableQuery,
+			});
+
+			await this.add('Accounts_Password_Policy_AtLeastOneLowercase', true, {
+				type: 'boolean',
+				enableQuery,
+			});
+
+			await this.add('Accounts_Password_Policy_AtLeastOneUppercase', true, {
+				type: 'boolean',
+				enableQuery,
+			});
+
+			await this.add('Accounts_Password_Policy_AtLeastOneNumber', true, {
+				type: 'boolean',
+				enableQuery,
+			});
+
+			await this.add('Accounts_Password_Policy_AtLeastOneSpecialCharacter', true, {
+				type: 'boolean',
+				enableQuery,
+			});
+		});
+
+		await this.section('Password_History', async function () {
+			await this.add('Accounts_Password_History_Enabled', false, {
+				type: 'boolean',
+				i18nLabel: 'Enable_Password_History',
+				i18nDescription: 'Enable_Password_History_Description',
+			});
+
+			const enableQuery = {
+				_id: 'Accounts_Password_History_Enabled',
+				value: true,
+			};
+
+			await this.add('Accounts_Password_History_Amount', 5, {
+				type: 'int',
+				enableQuery,
+				i18nLabel: 'Password_History_Amount',
+				i18nDescription: 'Password_History_Amount_Description',
+			});
+		});
+	});
+
+await settingsRegistry.addGroup('OAuth', async function () {
```

Comment:
Any reason not to have this inside a function as well?

Reply:
Good catch! This should be on Oauth file 😬